nixos/modules/services/web-apps/monica.nix

```diff
@@ -16,7 +16,8 @@ let
   user = cfg.user;
   group = cfg.group;
 
-  php = lib.getExe pkgs.php83;
+  phpPackage = cfg.phpPackage;
```

Comment:
I think this variable can be safely removed (to avoid confusion later on), and we can use `cfg.phpPackage` directly.

Reply:
done